clearing-house/src/lib.rs

```diff
@@ -2091,164 +1822,170 @@ pub mod pallet {
                     // Add PnL
                     equity.try_add_mut(&value.try_sub(&position.quote_asset_notional_amount)?)?;
                     // Add unrealized funding
-                    equity.try_add_mut(&<Self as Instruments>::unrealized_funding(
-                        &market, position,
-                    )?)?;
+                    equity.try_add_mut(&Self::unrealized_funding(&market, position)?)?;
                 }
             }
 
             Ok(equity >= min_equity)
         }
     }
 
-    // Helper functions - low-level functionality
+    // Funding helpers
```

Comment:
For consistency’ sake, I'd suggest sticking with `Helper functions - <something>`, similar to what was done [here](https://github.com/escher-exchange/escher/pull/12/files#diff-0b280c40791e05059aac1ea15132c2a3d62c2b65e187793dec52583352181e1dR1698)